docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,67 @@
+# Migrate from other CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea.
+
+NOTE: The following is a list of CNIs and versions we have verified
+the migration process, for your reference. CNIs and versions that are
+not listed probably work, please create an issue if you run into
+problems when migrating to Antrea.
+
+| CNI     | Version |
+|---------|---------|
+| Calico  | v3.26   |
+| Flannel | v0.22.0 |
+
+With the help of migration [script](../hack/migrate-to-antrea.sh),
+the migration process is fully automated. You can use the following
+command to start the migration:
+
+```bash
+./hack/migrate-to-antrea.sh
+```
+
+More details about the migration process are described in the following
+sections. And you can learn how to use the script by running
+`./hack/migrate-to-antrea.sh --help`.
+
+The migration process is divided into two steps:
+
+1. Install Antrea in the cluster.
+2. Restart all Pods in the cluster in-place.
+
+## Install Antrea
+
+After the requirement check, the script will install Antrea in the cluster
+by the following command:
+
+```bash
+kubectl apply -f ${ANTREA_MANIFEST} --kubeconfig $KUBECONFIG
+```
+
+## Restart Pods
+
+After Antrea is installed in the cluster, the script will restart all Pods in
+the cluster in-place by deploying a DaemonSet named `antrea-migrator`,
+which will run a Pod on each Node. The `antrea-migrator` Pod will kill all Pods'
+containerd tasks on each Node, and the containerd tasks will be restarted by
+the containerd service. In this way, all Pods in the cluster will be restarted
+in-place and will not `be rescheduled and recreated.
+
+The restart result is will be similar as follows, every Pod's `RESTARTS` will
+be increased by 1:
+
+```bash
+$ kubectl get pod -o wide
+NAME                               READY   STATUS    RESTARTS      AGE    IP          NODE          NOMINATED NODE   READINESS GATES
+migrate-example-6d6b97f96b-29qbq   1/1     Running   1 (24s ago)   2m5s   10.10.1.3   test-worker   <none>           <none>
+migrate-example-6d6b97f96b-dqx2g   1/1     Running   1 (23s ago)   2m5s   10.10.1.6   test-worker   <none>           <none>
+migrate-example-6d6b97f96b-jpflg   1/1     Running   1 (23s ago)   2m5s   10.10.1.5   test-worker   <none>           <none>
+```
+
+## Clean up old CNI
+
+After all Pods are restarted, Antrea will take over the network management and
+IPAM, but the old CNI may still be installed in the cluster. There might be
+static route or iptable rule conflicts between Antrea and the old CNI. So you
+need to clean up the old CNI manually. 
```

Comment:
```suggestion
IPAM, but the old CNI may still be running in the cluster. There might be
static routes or iptable rules conflicts between Antrea and the old CNI. So you
need to clean up the old CNI manually after the migration is finished. 
```

Should we highlight that the cluster should be paused and not to schedule any new Pods/Services? otherwise, it will have new stale rules after migration since the old CNI is still running.